Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1218 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
+# Textile language:         https://textile-lang.com/
+# Textile on GitHub:        https://github.com/textile
+# Textile specifications:   https://github.com/textile/textile-spec
+# Sublime syntax docs:      http://www.sublimetext.com/docs/syntax.html
+
 name: Textile
 scope: text.html.textile
+version: 2
```

Comment:
The way `html` is included in `main` and the main scope indicate extending HTML would be a proper choise here.

```suggestion
version: 2

extends: Packages/HTML/HTML.sublime-syntax
```

It would actually make it easier for template languages, which want to extend Textile, to also inject their patterns into HTML tags.

You could re-use HTML's `merge-conflict-markers`.

This change may require some smaller changes to current `html` context as it is the main one of HTML.sublime-syntax. Maybe further action needs to be taken to avoid possible context naming conflicts.

_FWIW, Markdown just can't be an extension of HTML as it has odd parsing rules for HTML blocks. It doesn't seem to be the case for textile?_